selfdrive/sensord/sensors/lsm6ds3_accel.cc

```diff
@@ -113,10 +114,14 @@ bool LSM6DS3_Accel::get_event(cereal::SensorEventData::Builder &event) {
   float y = read_16_bit(buffer[2], buffer[3]) * scale;
   float z = read_16_bit(buffer[4], buffer[5]) * scale;
 
+  auto event = msg.initEvent().initAccelerometer();
   event.setSource(source);
   event.setVersion(1);
   event.setSensor(SENSOR_ACCELEROMETER);
   event.setType(SENSOR_TYPE_ACCELEROMETER);
+  if (ts != 0) {
```

Comment:
is this check necessary? the timestamp should default to 0 anyway

Reply:
we could remove the `has_interrupt_enabled` if too, lsm will always run on interrupt

Reply:
yeah let's clean this up